### PR TITLE
Fix debugger and add mac configuration

### PR DESCRIPTION
**Pull request by DaeunYim**

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -4,14 +4,10 @@
  *--------------------------------------------------------------------------------------------*/
 'use strict';
 
-import * as nls from 'vscode-nls';
-
```
Comment (DaeunYim) [inline src/constants.ts:8]: nls.loadMessageBundle had issue but wasn't able to figure out immediately, As we are not supporting multi languages  at the moment, disabling this error causing localize

File: package.json [hunk 2]
```diff
@@ -13,6 +13,10 @@
   "activationEvents": [
     "*"
   ],
+  "engines":{
+    "vscode": "*",
+    "azdata": "*"
+  },
```
Comment (nasc17) [inline package.json:19]: Why was this added back in?

Comment (DaeunYim) [reply package.json:19]: Turned out packaging requires these engines
